src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala

```diff
@@ -1291,17 +1292,20 @@ object Solicitor_PRL_C100 {
         .get("/data/internal/cases/${caseId}")
         .headers(Headers.commonHeader)
         .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json")
-        .header("x-xsrf-token", "${XSRFToken}"))
+        .header("x-xsrf-token", "${XSRFToken}")
+        .check(jsonPath("$.state.id").is("AWAITING_SUBMISSION_TO_HMCTS")))
```

Comment:
You'll also need an additional check when you view the case to check that the latest event was added to the case successfully. e.g. .check(jsonPath("$.events........))

Reply:
Done